### DISCUSSION TITLE
Example of using Graphics class

### DISCUSSION THREAD

**Discussion by Drastef31** [ansys/pydyna#386] [Q&A] [answered]
Hello all,

Does anyone manage to make work the Graphics class and to display some objects ? Would you kindly share an example because I have been stuck on it for days ?
Thank you in advance

**Comment by RobPasMue** [comment]
Pinging @ansys/pydyna team

**Comment by Drastef31** [comment, marked as answer]
Finally, I answer to myself...

It worked with no more than something like that 👍

mymodel = Model(solution.stub)
graph = Graphics(mymodel, False)
graph()

but the server needed to be updated at last version. Once done, it worked perfectly.
